### PR TITLE
Show both owner login and repo name in list

### PR DESCRIPTION
**Pull request by raeesbhatti**
Related issue: https://github.com/desktop/desktop/issues/1533

### CODE REVIEW

File: app/src/ui/repositories-list/repository-list-item.tsx [hunk 1]
```diff
@@ -94,7 +91,7 @@ export class RepositoryListItem extends React.Component<
       : path
 
     let prefix: string | null = null
-    if (this.props.needsDisambiguation && gitHubRepo) {
+    if (gitHubRepo) {
       prefix = `${gitHubRepo.owner.login}/`
```
Comment (shiftkey) [inline app/src/ui/repositories-list/repository-list-item.tsx]: This prefix is used to show the owner in a slightly lighter colour. I'm not sure how the "never disambiguate" change here should impact this:

**Comment by shiftkey** [comment]
@raeesbhatti thanks for opening up this PR. This is something I was thinking about recently as a way to add more context, but I'm going to let this sit for a bit and gather feedback.

Some additional context/history:

 - the goal for using disambiguation here was that we believed we should only show as much as necessary, to avoid cluttering the repository list. For users who mostly work in the same organization, we felt that always showing that organization name felt like redundant information.
 - this disambiguation doesn't help when you have multiple copies of the same GitHub repository and want to differentiate between then. This change to always show the owner doesn't really address that either, and that's okay. The best issue tracking that is probably #1929.
 - we might have to suck it up and investigate more complex groupings (remote -> organization) and how filtering affects this.

**Comment by raeesbhatti** [comment]
@billygriffin My apologies for a late reply. The main reason for implementing this change was to "disambiguate" ownership of repos named like "examples", "sdk", "backend", "site". My work (like many others) spans across repos from multiple orgs, and different workstations have different locally cloned repos; having a repo named "backend" is not very intuitive atm. The existing logic for disambiguation requires two repos with same name to trigger which means that previously mentioned repo names would still not be clear.
The change does not affect any styling or positioning.

**Comment by shiftkey** [comment]
@raeesbhatti thanks for the feedback. For the moment I'm going to close this out as we've not really dug into the underlying issue, and I'd love to explore the problem space better. So please follow along with #1533 in the meantime.

**Pull request closed without merging**